services/static-webserver/client/source/boot/index.html

```diff
@@ -20,12 +20,22 @@
   <meta name="msapplication-tap-highlight" content="no"/>
 
   <!-- Disable chrome translation requests and automatic phone number linking -->
-  <meta name="google" value="notranslate"/>
+  <meta name="google" content="notranslate"/>
   <meta name="format-detection" content="telephone=no"/>
 
-  <!-- Shortcut icon setup -->
-  <link rel="shortcut icon" type="image/png" href="${resourcePath}osparc/favicon-osparc.png"/>
-  <link rel="mask-icon" href="${resourcePath}osparc/favicon-osparc.png"/>
+  <!-- Manifest -->
+  <!-- <link rel="manifest" href="/manifest.json"> - dynamically added -->
+
+  <!-- Safari Pinned Tab Icons -->
+  <link rel="mask-icon" href="" color="">
```

Comment:
Please keep the default favicons

Reply:
Am I right to assume you are referring to the mask-icon or are you talking about the shortcut icon?